apps/src/lab2/views/dialogs/GenericAlertDialog.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import GenericDialog, {GenericDialogProps} from './GenericDialog';
+
+export type GenericAlertDialogProps = Required<
+  Pick<GenericDialogProps, 'title'>
+> &
+  Pick<GenericDialogProps, 'message'>;
+
+const GenericAlertDialog: React.FunctionComponent<GenericAlertDialogProps> = ({
+  title,
+  message,
+}) => <GenericDialog title={title} message={message} />;
+
+export default GenericAlertDialog;
```

Comment:
All the dialogs wrapper around `GenericDialog` and hand through the props they've received. This keeps all UI centralized in the one component and much easier to change.